fluent-package/yum/systemd-test/downgrade-to-v4.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
+
+set -exu
+
+. $(dirname $0)/commonvar.sh
+
+# Install v4
+case ${distribution} in
+    amazon)
+	curl -fsSL https://toolbelt.treasuredata.com/sh/install-amazon2-td-agent4.sh | sh
+	;;
+    *)
+	curl -fsSL https://toolbelt.treasuredata.com/sh/install-redhat-td-agent4.sh | sh
+	;;
+esac
+
+# Not auto started
+(! systemctl status --no-pager td-agent)
+
+# Start service to generate log
+sudo systemctl enable --now td-agent
+sudo systemctl stop td-agent
+
+# Install the current
+sudo $DNF install -y \
+    /host/${distribution}/${DISTRIBUTION_VERSION}/x86_64/Packages/fluent-package-[0-9]*.rpm
+
+sudo systemctl daemon-reload
+sudo systemctl enable --now fluentd
+sudo systemctl stop fluentd
+
+# Remove td-agent.service alias not to conflict with v4
+sudo systemctl disable td-agent
+
+sudo $DNF remove -y fluent-package
+
+sudo rm -f /var/log/td-agent
+sudo rm -f /etc/td-agent
+sudo mkdir -p /var/log/td-agent
+sudo chown td-agent:td-agent /var/log/td-agent
+
+sudo mv /var/log/fluent/* /var/log/td-agent/
```

Comment:
For rpm packages, the symlinks for compatibility are automatically removed.
So, we don't need to remove them.

```suggestion
sudo $DNF remove -y fluent-package
(! test -e /etc/td-agent)
(! test -e /var/log/td-agent)
sudo mv /var/log/fluent/ /var/log/td-agent/
```

I have confirmed this works correctly on my fork CI (https://github.com/daipom/fluent-package-builder/actions/runs/7164750480)

Reply:
Surely no need to remove symlink in beforehand. Thanks.